include/experimental/__p0009_bits/config.hpp

```diff
@@ -231,7 +231,7 @@ static_assert(_MDSPAN_CPLUSPLUS >= MDSPAN_CXX_STD_14, "mdspan requires C++14 or
 #endif
 
 #ifndef MDSPAN_CONDITIONAL_EXPLICIT
-#  if MDSPAN_HAS_CXX_20 && !defined(_MDSPAN_COMPILER_MSVC)
```

Comment:
effectively this impacted some tests which didn't guard the convertibility tests correctly for C++20 & MSVC. But with current MSVC this just works anyway.